src/IECoreMaya/ToMayaMeshConverter.cpp

```diff
@@ -344,6 +346,123 @@ bool ToMayaMeshConverter::doConversion( IECore::ConstObjectPtr from, MObject &to
 		}
 	}
 
+	/// Add corners and edge creases
+
+	MUintArray cornerIdsMaya;
+	MDoubleArray cornerSharpnessMaya;
+
+	const IECore::IntVectorData *cornerIdsData = mesh->cornerIds();
+	const std::vector<int> cornerIds = cornerIdsData->readable();
+
+	if( !cornerIds.empty() )
+	{
+		const IECore::FloatVectorData *cornerSharpnessesData = mesh->cornerSharpnesses();
+		const std::vector<float> cornerSharpnesses = cornerSharpnessesData->readable();
+
+		for( size_t i = 0; i < cornerIds.size(); ++i )
+		{
+			cornerIdsMaya.append( cornerIds[i] );
+			cornerSharpnessMaya.append( cornerSharpnesses[i] );
+		}
+
+		s = fnMesh.setCreaseVertices( cornerIdsMaya, cornerSharpnessMaya );
+		if( !s )
+		{
+			IECore::msg( IECore::Msg::Warning, "ToMayaMeshConverter::doConversion", boost::format( "Failed to set crease vertices with message: %s" ) % s.errorString().asChar() );
+		}
+	}
+
+	MUintArray edgeIdsMaya;
+	MDoubleArray creaseSharpnessMaya;
+
+	const IECore::IntVectorData *creaseIdsData = mesh->creaseIds();
+	const std::vector<int> creaseIds = creaseIdsData->readable();
+
+	if( !creaseIds.empty() )
+	{
+		const IECore::FloatVectorData *creaseSharpnessesData = mesh->creaseSharpnesses();
+		const std::vector<float> creaseSharpnesses = creaseSharpnessesData->readable();
+
+		const IECore::IntVectorData *creaseLengthsData = mesh->creaseLengths();
+		const std::vector<int> creaseLengths = creaseLengthsData->readable();
+
+		// Cortex stores vertex ids to specify creases. Maya uses edge ids
+		// instead. The following handles the conversion.
+
+		MDagPath dag;
+		fnMesh.getPath( dag );
+		MItMeshVertex vertexIt( dag );
```

Comment:
Can we not use `vertexIt( mObj )` here? My understanding is that not all MObjects have an MDagPath (for instance, a mesh we're creating to pass out of a compute, rather than assign to a shape), so `dag` may not always be valid.

Reply:
Fixed in 80bf351. Thanks for pointing that out.